semanticdb/integration/src/main/scala/example/Synthetic.scala

```diff
@@ -44,4 +44,10 @@ class Synthetic {
     if a < b
   } yield a
 
+  // See https://github.com/scalameta/scalameta/issues/1749
+  import scala.math.Ordered.orderingToOrdered
```

Comment:
Just curious, can this import and `new F` be replaced with `val x = 42; (x, x)`? Would be nice to minimize the test case as much as possible